package.json

```diff
@@ -47,6 +47,7 @@
     "rollup-plugin-svelte": "7.1.2",
     "sass": "1.58.0",
     "sass-loader": "13.2.0",
+    "skiafy": "github:fallaciousreasoning/skiafy",
```

Comment:
So this `skiafy` that we forked[0] seems to be a conversion of the original skia repo[1] that works in node. It seems like it hasn't been maintained in 4 years. I suggest we take the current skiafy script and make the same node-compatibility changes as just put it in leo, as this is the only place we're using it at Brave and we don't need the added complexity of another fork that we won't look at for years.

[0] https://github.com/zhsoft88/skiafy
[1] https://github.com/evanstade/skiafy

Reply:
Moved in to Leo :smile: